code/processes/queryfeed.q

```diff
@@ -0,0 +1,49 @@
+// queryfeed proc script - subs to .usage.usage tables and publishes to query tickerplant
+
+// add connections to all procs for query tracking to be enabled
+.servers.CONNECTIONS:.servers.CONNECTIONS,exec distinct proctype from ("  SS         ";enlist csv) 0: hsym `$getenv `TORQPROCESSES where procname in subprocs;
```

Comment:
I think `subprocs` should have a default value in this script, similar to other TorQ scripts (I know it is defined in config, but typically we also have a default to fall back on in the main script itself)